core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -772,6 +772,9 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     // this behaves a little different than when remaining limit bytes is 0 and it's important to test it
     this.consumerConfig.setProperty(ConsumerConfig.FETCH_MAX_BYTES_CONFIG, "500")
     this.consumerConfig.setProperty(ConsumerConfig.MAX_PARTITION_FETCH_BYTES_CONFIG, "100")
+
+    // set the 'max.poll.interval.ms' larger (default is 6 seconds), to avoid the consumer got kicked out during sending records
```

Comment:
This comment can be more concise. Something like:

`Avoid a rebalance while the records are being sent (the default is 6 seconds)`

Reply:
Updated.